src/msw/listctrl.cpp

```diff
@@ -880,6 +884,25 @@ bool wxListCtrl::GetItem(wxListItem& info) const
     return success;
 }
 
+// Check if the item is visible
+bool wxListCtrl::IsVisible(long item) const
+{
+    bool result = ::SendMessage( GetHwnd(), LVM_ISITEMVISIBLE, (WPARAM) item, 0 );
```

Comment:
```suggestion
    bool result = ::SendMessage(GetHwnd(), LVM_ISITEMVISIBLE, (WPARAM) item, 0) != 0;
```
Otherwise you're going to get warnings from MSVC about int-to-bool conversion.